src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import styles from './courseCard.module.scss';
+import { Button } from '../../../../common/Button/Button';
+import { BUTTON_TEXT_SHOW_COURSE } from '../../../../constants';
+import {
+	dateConvert,
+	getAutors,
+	timeConvert,
+} from '../../../../helpers/getCourseInfo';
+
+const CourseCard = ({
+	title,
+	description,
+	creationDate,
+	duration,
+	authors,
+}) => {
+	return (
+		<div className={styles.wrapper}>
+			<div className={styles.courseDesc}>
+				<h2>{title}</h2>
+				<p>{description}</p>
+			</div>
+			<div className={styles.courseInfoWrapper}>
+				<div className={styles.courseInfo}>
+					<div>
+						<span>Autors: </span>
+						<p>{getAutors(authors).join(', ')}</p>
```

Comment:
Criteria "if all authors' names do not fit on one line, then the extra text should be cut off and '...' should be added at the end of line" is not met

Reply:
fixed, thanks